ext/background/const.js

```diff
@@ -12,6 +12,10 @@ const MERGE_BEHAVIOR = {
   REPLACE_EXISTING: 'REPLACE_EXISTING',
 };
 
+// Currently inelegant, but this is a quick way of tracking what new providers
+// have been added in a given update.
+const NEW_PROVIDERS = ['clubhouse'];
```

Comment:
Is there anyplace we can leave a comment to remind us to update this when new providers are added?

Reply:
Not at the moment. I think I need to update the repo docs. I'll make sure to do that before v2.3.0 goes out.